drivers/google/google_test.go

```diff
@@ -0,0 +1,153 @@
+package google
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/codegangsta/cli"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDriverName(t *testing.T) {
```

Comment:
This test really seems quite unnecessary to me.  Why test a function which simply returns a hardcoded string?